core/src/main/java/org/infinispan/stream/impl/AbstractCacheStream.java

```diff
@@ -66,7 +67,7 @@
  * @param <S> The stream interface
  */
 public abstract class AbstractCacheStream<T, S extends BaseStream<T, S>, S2 extends S> implements BaseStream<T, S> {
-   protected final Log log = LogFactory.getLog(getClass());
+   protected static final Log log = LogFactory.getLog(MethodHandles.lookup().lookupClass());
```

Comment:
Actually this line can be completely removed (I missed removing it on a refactor). This is already using the `getLog` approach I mentioned on the off heap one.